exports/lint-configs/.clang-format

```
@@ -56,6 +57,7 @@ BreakBeforeBinaryOperators: "All"
 BreakBeforeBraces: "Custom"
 BreakBeforeConceptDeclarations: "Always"
 BreakBeforeInlineASMColon: "OnlyMultiline"
+BreakBeforeTemplateCloser: false
```

```suggestion
BreakBeforeTemplateCloser: true
```

To be more consistent with our formatting of other syntax.